nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/main/java/org/nuxeo/ecm/platform/comment/impl/PropertyCommentManager.java

```diff
@@ -73,6 +74,22 @@
 
     protected static final String COMMENT_NAME = "comment";
 
+    /**
+     * Counts how many comments where made on a specific document.
+     *
+     * @since 11.1
+     */
+    protected static final String QUERY_GET_COMMENTS_COUNT_FOR_DOCUMENT_BY_COMMENT_ANCESTORS = //
+            "SELECT " + ECM_UUID + " FROM Comment WHERE " + COMMENT_ANCESTOR_IDS + "/* = '%s'";
```

Comment:
Why not using a `COUNT` query?

Reply:
Mongo doesn't support projections. Tried to count with SQL but it didn't find the docs.
Here the mongo error:
```
2020-01-17 13:03:58,042 [main] WARN  [TransientStorageComponent] Missing configuration for default transient store, using in-memory
org.nuxeo.ecm.core.query.QueryParseException: Projection not supported: count(ecm:uuid)

```